dask/utils.py

```diff
@@ -1484,7 +1484,19 @@ def format_time(n: float) -> str:
     '123.45 us'
     >>> format_time(123.456)
     '123.46 s'
+    >>> format_time(1234.567)
+    '20m 34s'
+    >>> format_time(12345.67)
+    '3h 25m'
```

Comment:
Totally open to other formats here

Reply:
I have no issues with this format but perhaps this could be configurable?

[This implementation looks nice](https://stackoverflow.com/a/42320260/1003288) and allows you to pass an optional format string.